stdlib/smtplib.pyi

```diff
@@ -125,16 +125,16 @@ class SMTP:
     @overload
     def auth_cram_md5(self, challenge: None = ...) -> None: ...
     @overload
-    def auth_cram_md5(self, challenge: bytes) -> str: ...
-    def auth_plain(self, challenge: bytes | None = ...) -> str: ...
-    def auth_login(self, challenge: bytes | None = ...) -> str: ...
+    def auth_cram_md5(self, challenge: ReadableBuffer) -> str: ...
+    def auth_plain(self, challenge: ReadableBuffer | None = ...) -> str: ...
```

Comment:
The `challenge` parameter doesn't look like it's actually used for this one

Reply:
The next one doesn't really use it either, I figured I should be consistent with `auth_cram_md5`.

Reply:
Yeah, that makes sense, I figured as much!